### PR TITLE
UI tweak - display planet wheel only for real planets

### PR DESCRIPTION
**Pull request by rrobgill**
This patch hides the planet wheel and intro text when viewing the
encyclopedia planet suitability page for gas giants and asteroids.

Related Issue #3003

Signed-off-by: Rob Gill <rrobgill@protonmail.com>

### CODE REVIEW

File: UI/EncyclopediaDetailPanel.cpp [hunk 1]
```diff
@@ -2680,8 +2680,10 @@ namespace {
             detailed_description += "\n";
         }
 
-        detailed_description += UserString("ENC_SUITABILITY_REPORT_WHEEL_INTRO")
-                                + "<img src=\"encyclopedia/EP_wheel.png\"></img>";
+        if (planet->Size() < SZ_ASTEROIDS) {
```
Comment (adrianbroher) [inline UI/EncyclopediaDetailPanel.cpp]: The correct semantic would be to check against `Type`, not `Size` as this check also covers `SZ_NOWORLD`.

Also this needs to exclude `INVALID_PLANET_TYPE`.

Comment (rrobgill) [reply UI/EncyclopediaDetailPanel.cpp]: fixed in a1a5683

**Comment by geoffthemedio** [comment]
@rrobgill feel free to / please add yourself to credits.xml in your next submission

**Comment by Vezzra** [comment]
> @rrobgill feel free to / please add yourself to credits.xml in your next submission

Just did that for him: fdccaab6a782bdd0f4e96b4cefedc3e296675f0a

**Pull request merged**